packages/nimble-blazor/CONTRIBUTING.md

```diff
@@ -50,7 +50,12 @@ public partial class NimbleButton : ComponentBase
 ```
     <nimble-button>@ChildContent</nimble-button>
 ```
-- Code style conventions are enforced by the [NI C# Style Guide](https://github.com/ni/csharp-styleguide) 
+- Always add an `AdditionalAttributes` parameter that captures unmatched values, so that attributes not declared explicitly can still be passed along to the Nimble element:
```

Comment:
```suggestion
- Always add an `AdditionalAttributes` parameter that captures unmatched values, so that attributes not declared explicitly, such as the common `class` or `id` attributes, can still be passed along to the Nimble element:
```